sgl-kernel/benchmark/bench_fp8_gemm.py

```diff
@@ -0,0 +1,63 @@
+import torch
+import torch.nn.functional as F
+import triton
+from sgl_kernel import fp8_scaled_mm as sgl_scaled_mm
+from vllm._custom_ops import cutlass_scaled_mm as vllm_scaled_mm
+from vllm._custom_ops import scaled_fp8_quant as vllm_scaled_fp8_quant
+
+
+@triton.testing.perf_report(
```

Comment:
Can we add a CuBlas version to compare, refer to https://github.com/vllm-project/vllm/blob/main/benchmarks/cutlass_benchmarks/w8a8_benchmarks.py .